.evergreen/config.yml

```diff
@@ -317,3 +323,12 @@ buildvariants:
       - name: test-haystack-fulltext-local
       # TODO: INTPYTHON-465
       # - name: test-haystack-fulltext-remote
+
+  - name: test-langchaingo-ubuntu
+    display_name: LangchainGo Ubuntu2204
+    expansions:
+      DIR: langchaingo-golang
+    run_on:
+      - ubuntu2204-small
```

Comment:
How come you've changed the host? It was a set and forget thing. Does rhel87-small not have your go version? Should we consider updating all?

Reply:
The tests rely on `docker`, which isn't available on the RHEL hosts.